shared/native/notification-listeners.desktop.js

```diff
@@ -32,6 +33,10 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
       dispatch(bootstrap())
       response.result()
     },
+    'keybase.1.NotifySession.clientOutOfDate': ({upgradeTo, upgradeURI, upgradeMsg}) => {
+      const body = upgradeMsg || `Please update to ${upgradeTo} by going to ${upgradeURI}`
+      NotifyPopup('Client out of date!', {body})
```

Comment:
updated. i really dislike this notify param. why even bother doing this? also the fact that its typed as 'any' means its hard to understand. i'm changing it so its like the rest but i think this is an anti-pattern